src/noSuspiciousCommentRule.ts

```diff
@@ -6,7 +6,7 @@ import { ExtendedMetadata } from './utils/ExtendedMetadata';
 
 const FAILURE_STRING: string = 'Suspicious comment found: ';
 const SUSPICIOUS_WORDS = ['BUG', 'HACK', 'FIXME', 'LATER', 'LATER2', 'TODO'];
-const FAILURE_STRING_OPTION: string = '\nTo disable this warning, the comment should include one of the following regex: ';
+const FAILURE_STRING_OPTION: string = 'To disable this warning, the comment should include one of the following regex: ';
```

Comment:
I couldn't figure out how to get errors across lines to work so I just removed the endline...